src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -223,14 +223,10 @@ private static ProjectItemGroupElement CreateItemGroup(Project project, string f
         {
             var packageVersion = packageDependency.VersionRange.OriginalString ??
                 packageDependency.VersionRange.MinVersion.ToString();
-            var packageMetadata = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
-            { { VERSION_TAG, packageVersion } };
 
-            // Currently metadata is added as a metadata. As opposed to an attribute
-            // Due to https://github.com/Microsoft/msbuild/issues/1393
+            var item = itemGroup.AddItem(PACKAGE_REFERENCE_TYPE_TAG, packageDependency.Id);
+            item.AddMetadata(VERSION_TAG, packageVersion, expressAsAttribute: true);
 
-            itemGroup.AddItem(PACKAGE_REFERENCE_TYPE_TAG, packageDependency.Id, packageMetadata);
-            itemGroup.ContainingProject.Save();
```

Comment:
This was saving the project file containing the item group. But Thats done [here](https://github.com/NuGet/NuGet.Client/blob/b2475eacc74bd27ea0d18bb50879e905a4b7787b/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs#L161) in the caller, so it can be removed from here.